src/main/java/globals/Colors.java

```diff
@@ -0,0 +1,24 @@
+package globals;
+
+import java.awt.*;
+
+public class Colors {
+
+    public static final Color HEADER_COLOR                  = new Color(29, 31, 48);
+
+    public static final Color BACKGROUND_COLOR              = new Color(108, 125, 135);
+
+
+    public static final Color PANELS_COLOR                  = new Color(73, 106, 122);
+    public static final Color TEXT_COLOR                    = new Color(220, 224, 229);
+
+    public static final Color BUTTON_COLOR                  = new Color(23, 13, 0);
+    public static final Color BUTTON_COLOR_HOVER            = new Color(23, 13, 0);
+
+
+
+    public static final Color OPTIONS_BUTTON_COLOR          = new Color(255, 255, 255);
+    public static final Color OPTIONS_BUTTON_HOVER_COLOR    = new Color(240, 240, 240);
+
+
+}
```

Comment:
Really love the definition of colours in a different class. I know this is not really criticism, but it really makes me happy when i see it.